### PR TITLE
add several GitHub/Slack, Slack/Slack, GitHub/Github applets

### PR DESCRIPTION
**Pull request by ghost**
- discussion-notifier
- label-notifier.rs
- release-notifier.rs
- repository-notifier.rs
- workflow-job-notifier.rs
- fork-notifier.rs

- lucky-draw.rs

- auto-issue-comment.rs

### CODE REVIEW

File: slack/slack/lucky-draw/rust/src/lucky-draw.rs [hunk 1]
```diff
@@ -0,0 +1,27 @@
+#[allow(unused_imports)]
+use wasmedge_bindgen::*;
+use wasmedge_bindgen_macro::*;
+use rand::{thread_rng, Rng};
+
+#[wasmedge_bindgen]
+pub fn run(s: String) -> String {
+    let mut rng = thread_rng();
+
+    match s.as_str() {
```
Comment (ho-229) [inline slack/slack/lucky-draw/rust/src/lucky-draw.rs]: You can use `to_lowercase()` instead of enumerate the case.

**Review by ho-229 — CHANGES_REQUESTED** [review]

File: slack/slack/lucky-draw/rust/src/lucky-draw.rs (hunk 1 above)
Comment (ghost) [reply slack/slack/lucky-draw/rust/src/lucky-draw.rs]: sure, thanks

**Comment by ghost** [comment]
done

File: slack/slack/lucky-draw/rust/src/lucky-draw.rs (hunk 1 above)
Comment (ho-229) [reply slack/slack/lucky-draw/rust/src/lucky-draw.rs]: No changes here yet

**Review by ho-229 — APPROVED** [review]

**Pull request merged**